llvm/test/CodeGen/AMDGPU/load-store-opt-dlc.mir

```diff
@@ -1,4 +1,5 @@
 # RUN: llc -mtriple=amdgcn -mcpu=gfx1010 -run-pass=si-load-store-opt -verify-machineinstrs -o - %s | FileCheck %s
+# RUN: llc -mtriple=amdgcn -mcpu=gfx1010 -passes=si-load-store-opt -verify-machineinstrs -o - %s | FileCheck %s
```

Comment:
Verify is default when using new pass manager.

Reply:
I think the default should be pre and post verify the MIR, and the flag should be interpreted as run after each pass

Reply:
@paperchalice I wonder why the verify is default when using NPM? Shouldn't the behavior be consistent with the legacy PM, at least this early stage?

Reply:
Because verify is no longer a part in pass pipeline when using new pass manager. New pass manager introduces `PassInstrumentation`, verify now is now part of `StandardInstrumentations::Verify` which support both kinds of IRs. This can be disabled by option `--disable-verify`, but the doc string is not updated...

Reply:
we should match `opt`'s `-disable-verify` and `-verify-each`

Reply:
> we should match `opt`'s `-disable-verify` and `-verify-each`

Now in https://github.com/llvm/llvm-project/pull/106665.